larray_editor/arraymodel.py

```diff
@@ -125,7 +125,8 @@ class LabelsArrayModel(AbstractArrayModel):
     font : QFont, optional
         Font. Default is `Calibri` with size 11.
     """
-    def __init__(self, parent=None, data=None, readonly=False, font=None):
+    def __init__(self, parent=None, data=None, readonly=False, font=None, orientation=Qt.Horizontal):
```

Comment:
If it turns out to be indeed necessary to differentiate on the orientation (I am unsure about this), then having two different subclasses of LabelsArrayModel depending on the orientation would make things cleaner.